client/controller/shared.js

```diff
@@ -35,25 +37,62 @@ export function makeLayoutMiddleware( LayoutComponent ) {
 	};
 }
 
+function loadCSS( cssUrl, elementId = null ) {
```

Comment:
Because it's simpler to have a function that handles both cases so we can test both cases easily when working on this PR. Since this is a very common case we might want to reuse this function later.